### PR TITLE
Nisp 1813 - Refactor nispMain.scala.html and main.scala.html

### PR DESCRIPTION
**Pull request by roundrobin99**
I have merged nispMain into main. 

I have also modified browserTitle to an [option], as all other strings are, keeping it consistent.

### CODE REVIEW

**Comment by roundrobin99** [comment]
Hmmm...it looks one test failed although I am not sure how it relates to the changes I made. @dspork can you have a look ?

**Comment by roundrobin99** [comment]
I have added the service header in the main account page @prashantchoudhari

File: test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala [hunk 1]
```diff
@@ -199,7 +199,7 @@ class AccountControllerSpec extends UnitSpec with MockitoSugar with BeforeAndAft
 
       "return COPE page for contracted out (B) user" in {
         val result = MockAccountController.showCope()(authenticatedFakeRequest(mockUserIdContractedOut))
-        contentAsString(result) should include ("You were contracted out ")
+        contentAsString(result) should include ("Your COPE estimate is")
```
Comment (dspork) [inline test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala]: Why was this test changed?

Comment (roundrobin99) [inline test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala]: The text that we were testing had been changed, and test was not passing, so I used some other content.

Comment (dspork) [inline test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala]: How can the text have changed unless one of your commits did that, is this not a broken test?

**Comment by roundrobin99** [comment]
Reverted COPE test to what it was.

**Comment by roundrobin99** [comment]
@prashantchoudhari @InduTest I have made the necessary changes so this is ready to test

**Comment by dspork** [comment]
@InduTest If this does not pass regression then the refactoring has broken something and needs fixed.  You should not need to change any xpath for this request.

**Comment by prashantchoudhari** [comment]
@dspork  - please review.

**Comment by dspork** [comment]
I've just realised looking at this that `h1classes` does literally nothing

**Comment by prashantchoudhari** [comment]
I think some of them are used example "welcome-msg"

**Comment by roundrobin99** [comment]
We will need to review this, as we might not need it anymore. Perhaps another story in the backlog ?

**Comment by dspork** [comment]
Maybe should put this on hold while we sort out the h1 classes

**Comment by roundrobin99** [comment]
@dspork, @InduTest, I think this can now be tested and merged.

**Comment by InduTest** [comment]
@dspork  I will run a regression and once all ok good to merge

**Comment by dspork** [comment]
Lets wait for https://github.com/hmrc/nisp-frontend/pull/137

**Comment by roundrobin99** [comment]
I will need to remove some code, so don't merge it yet. 
We need to merge NISP-1929 first.

**Comment by roundrobin99** [comment]
This is ready for testing.I have merged this branch with NISP-1929 so we can use the refactored main with the `H1classes` removed. We can just merge this branch to have both 1813 and 1929, or we can do it separately.

File: app/uk/gov/hmrc/nisp/views/nispMain.scala.html [hunk 2]
```diff
@@ -1,57 +0,0 @@
...
-    contentHeader = Some(contentHeader),
-    userLoggedIn = userLoggedIn,
-    showTitleHeaderNav = showTitleHeaderNav,
-    showBetaBanner = showBetaBanner,
-    pageScripts = pageScripts,
-    articleClasses = articleClasses,
-    analyticsAdditionalJs = analyticsAdditionalJs,
-    articleEnabled = articleEnabled) {
-        @if(pageTitle.isDefined) {
-        <h1 class="heading-large top-title-heading"> @pageTitle </h1>
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/nispMain.scala.html]: Why have we removed this functionality?
It seems a bit silly to have ```<h1 class="heading-large top-title-heading"> @pageTitle </h1>```in every template.

Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/views/nispMain.scala.html]: The only way I have found to include the title within the `article` section, is by doing it separately in each file. Not ideal I agree, but it works. If you have any suggestions on how to keep it, let me know.

File: app/uk/gov/hmrc/nisp/views/main.scala.html [hunk 3]
```diff
@@ -46,6 +47,9 @@
   <li><a href="@Constants.logoutUrl" role="button">Sign out</a></li>
 }
 
+@contentWithTitle =  {
+    @mainContent
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/main.scala.html]: Trying adding `@pageTitle` to here. It already seems to be setup to expand the article section.

Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/views/main.scala.html]: I tried this but it didn't do it, i don't know why. Do you want to try it ?

**Pull request merged**